tests/sshserver.pl

```diff
@@ -764,8 +769,8 @@ sub sshd_supports_opt {
 
 if ($^O eq 'MSWin32' || $^O eq 'cygwin' || $^O eq 'msys') {
     # convert MinGW drive paths to Cygwin drive paths
-    $identity_config =~ s/^\/(\w)\//\/cygdrive\/$1\//;
-    $knownhosts_config =~ s/^\/(\w)\//\/cygdrive\/$1\//;
+    chomp($identity_config = `cygpath -u -a $identity_config`);
+    chomp($knownhosts_config = `cygpath -u -a $knownhosts_config`);
```

Comment:
Hello,

There is a letter that came to me as a complete surprise, you may find it interesting, please read it here <http://tendaspingy.bennettleak.com/xycmmxuq>

All best, paf@frobbit.se